.github/workflows/get-installations.yaml

```diff
@@ -39,10 +42,18 @@ jobs:
       email: ${{ steps.repos.outputs.email }}
     steps:
       - id: new-repos
+        name: "Evaluate inputs"
         run: |
+          echo '::debug::repo: ${{ inputs.repo }}'
+          echo '::debug::newbies: ${{ inputs.newbies }}'
           {
             echo "new<<EOF"
-            echo '${{ inputs.newbies }}'
+            if [[ $(grep -c '/' <<< '${{ inputs.repo }}') -gt 0 ]]; then
+              echo '${{ inputs.repo }}' | \
+              jq -Rc '[. | split("/") | {owner:.[0], name:.[1]}]'
+            else
+              echo '${{ inputs.newbies }}'
+            fi
```

Comment:
`inputs.newbies` is expected to be a JSON list of objects like this:

```
[{"owner": "a", "repo": "arepo"}, {"owner": "b", "repo": "brepo"}]
```

This addition allows someone to specify `repo: a/arepo` in the workflow call to create a single-item JSON list.